packages/studio-plugin/src/sourcefiles/ComponentFile.ts

```diff
@@ -42,12 +47,29 @@ export default class ComponentFile {
     };
 
     const filepath = this.studioSourceFileParser.getFilepath();
-
+    const cssImports = getCssFilesFromDependencyTree(this.dependencyTree);
     return {
       kind: FileMetadataKind.Component,
       ...this.fileMetadataParser.parse(onProp),
       ...(acceptsChildren ? { acceptsChildren } : {}),
       filepath,
+      cssImports,
     };
   };
 }
+
+function getCssFilesFromDependencyTree(dependencyTree: Tree): string[] {
+  const cssFiles = Object.entries(dependencyTree).reduce(
+    (cssFiles, [filename, subDependencyTree]) => {
+      if (filename.includes(".css")) {
+        cssFiles.add(filename);
+      }
+      return new Set([
+        ...cssFiles,
+        ...getCssFilesFromDependencyTree(subDependencyTree),
+      ]);
+    },
+    new Set<string>()
+  );
+  return Array.from(cssFiles);
```

Comment:
nit: I slightly prefer the spread syntax here instead of `Array.from`. I think it's supposed to be a little faster